.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
> You mean it crashes even when it "runs" a test group in search of requested tagged tests? 

Yes even running busted -l to enumerate the tests causes this. Because this code is not a test and is right at the top of the file (but it does spawn Neovim for the checks).